Banking_App/src/main/java/com/cognologix/assignment/BankSystemAssignment/exception/InsufficientBalanceExceptionHandler.java

```diff
@@ -0,0 +1,4 @@
+package com.cognologix.assignment.BankSystemAssignment.exception;
+
+public class InsufficientBalanceExceptionHandler {
```

Comment:
is this getting used ??
not extending any exception

Reply:
Refactored this exception